CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log
```

Comment:
It should be a single word. Is this autogenerated? We need to send in a PR. 😛

Reply:
Yes, this is auto-generated file. Here it says so.
*This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*